rpc/internal/formats.ext.go

```diff
@@ -0,0 +1,50 @@
+package internal
+
+import "github.com/evergreen-ci/cedar/model"
+
+func (f DataFormat) Export() model.FileDataFormat {
```

Comment:
Can we remove this or is the assumption that it'll be removed in a different PR?

Reply:
We still need it for perf